ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -75,6 +75,7 @@ const EthOverview = ({ className }) => {
             <div className="eth-overview__primary-container">
               {balance ? (
                 <UserPreferencedCurrencyDisplay
+                  style={{ display: 'contents' }}
```

Comment:
Since this component has a `eth-overview__primary-balance` className, can you add this style to the stylesheet (`ui/components/app/wallet-overview/index.scss`) instead of inline within the component code?

Reply:
The `currency-display-component` class's display tag within the `UserPreferencedCurrencyDisplay` component overrides `display: contents` in `eth-overview__primary-balance`. This causes the balance to wrap in a separate div from the rest of the div styled with `eth-overview__primary-balance` (see image).

I considered adding `display:contents` to `currency-display-component` as well, but it would have affected the way other balances are rendered throughout the extension so I chose to add an element-specific style here

<img width="353" alt="Screenshot 2023-03-12 at 8 02 35 PM" src="https://user-images.githubusercontent.com/100321200/224599145-b5c3b53f-da11-4271-b0ed-62b9771550f5.png">